lib/jormungandr/test/manual/TrackBlockHeight.md

```diff
@@ -0,0 +1,62 @@
+# Track Block Height
+
+## Goal
+
+The goal of this test suite is to make sure that correct block height is being tracked and logged by the wallet for Jörmungandr node.
+
+## Prerequisites
+
+Start Jörmungandr genesis-praos self-node or connect to the testnet with passive or leader node.
+Start wallet backend on top of working Jörmungandr node.
+e.g.
+```
+cardano-wallet-jormungandr launch --genesis-block config/block-0.bin --secret config/pool-secret1.yaml --state-dir config/hff
+```
+or
+```
+jormungandr --genesis-block config/block-0.bin --config config/config.yaml --secret config/pool-secret1.yaml
```

Comment:
Neither of these commands would work out-of-the-box in the repo though? Why not refer to the `cardano-wallet-jormungandr launch --genesis-block lib/jormungandr/test/data/jormungandr/block0.bin --secret lib/jormungandr/test/data/jormungandr/secret.yaml`?